WordPress/WordPressTest/PagesListTests.swift

```diff
@@ -72,9 +72,7 @@ class PagesListTests: CoreDataTestCase {
         NSLog("\(pages.count) pages used in \(#function)")
 
         measure {
-            let pageTree = PageTree()
-            pageTree.add(pages)
-            let list = pageTree.hierarchyList()
+            let list = (try? PageTree.hierarchyList(of: pages)) ?? []
```

Comment:
I was going to suggest the following, which I feel would make the point at which the test might have failed.

```suggestion
            let list = try XCTUnwrap(try? PageTree.hierarchyList(of: pages)))
```

But I checked [the docs](https://developer.apple.com/documentation/xctest/xctestcase/3194264-measure) and I guess it's not possible to `throw` from the block given to `measure` 😞

Reply:
Yep, that's why I added an assertion below, to make sure we are measuring success cases, not failure cases.